src/main/java/com/iemr/hwc/controller/wo/LocationControllerWo.java

```diff
@@ -154,4 +157,40 @@ public String getLocDetailsBasedOnSpIDAndPsmIDNew(@RequestBody String comingRequ
         }
         return response.toString();
     }
+
+    @CrossOrigin
+    @ApiOperation(value = "Updating village coordinates collected by nurse", consumes = "application/json", produces = "application/json")
+    @RequestMapping(value = { "/update/geolocation/wo" }, method = { RequestMethod.POST })
```

Comment:
@ApiOperation(value = "Updating village coordinates collected by nurse") change to Update health and wellness center coordinates

Reply:
Done